README.md

```diff
@@ -1 +1,16 @@
-# CalendarioSemanal
\ No newline at end of file
+# SMITETracker
+## Descripción del problema
+Como comunidad competitiva del videojuego SMITE, tenemos un problema común, y es que no hay información fácilmente accesible de jugadores y equipos rivales. Nos gustaría tener alguna forma de acceder a las estadísticas más relevantes del equipo rival de una manera más eficiente de la que tenemos actualmente, que es ir jugador a jugador investigando. Queremos extraer las estadísticas que nos interesan de una web poco estructurada con datos irrelevantes para nosotros, para poder obtener un resumen de la información que realmente nos interesa de los jugadores a los que nos enfrentamos.
+
+## Historias de usuario
+
+<a href="/docs/historias_usuario.md">Historias de usuario</a>
```

Comment:
Te dejo aquí también un comentario de forma... en markdown existe una manera de insertar links _más amable_ que realizarlos inyectando HTML...

`[texto_del_link](the/link)`

que sustituiría a `<a href="the/link">texto_del_link</a>`